esp/esp/themes/theme_data/fruitsalad/scripts/mit_usersearch.js

```diff
@@ -75,12 +75,17 @@ if (currentPrograms && currentPrograms.forEach) {
                 displayName: 'Class Search' +
```

Comment:
I personally think it might be nice to conserve a bit of space and condense the class search/manage sections into one section that would basically be the manage section of the tool bar with the last line saying "class search: _textbox_". I think this would make sense since both things are attached to the same program and then there would only be one section of the admin bar per current program instead of two:

![image](https://user-images.githubusercontent.com/25392075/77974351-d58a7980-72c4-11ea-9306-d66c07e29ccc.png)

Reply:
I think the placeholder text is self-explanatory, so probably just need to remove the header and include the search box in the same box as the other links.

Reply:
Fixed this @kkbrum 
Note that this file mit_usersearch.js isn't used anymore and is replaced by admin_bar.js